zerver/openapi/zulip.yaml

```
@@ -11346,10 +11346,10 @@ paths:
                           old_unreads_missing:
                             type: boolean
                             description: |
-                              True if this data set was truncated because the user has too many
+                              If `true`, this data set was truncated because the user has too many
```

nit:
```suggestion
                              Whether this data set was truncated because the user has too many
```

With "If `true`," I feel like I'm waiting for a later sentence to answer the question of what it means when it's false; and taken literally, it never does answer that. (It could be that when this is `false`, the data set sometimes also was truncated.) By saying "whether", we can cover both the true and false cases at once.

The old version already had a version of this problem (so "Whether" would already have been an improvement over "True if"), but it feels less so. Perhaps partly because the old version could have been fixed by a very small edit of reading "if" as "just if", or "iff", whereas the "If `true`," feels like it's explicitly setting up to separately say "If `false`,".

---

This makes sense. I've updated for your edit above: `Whether this data set was truncated ...`.